backend/imagekit/imagekit_test.go

```diff
@@ -0,0 +1,18 @@
+package imagekit
```

Comment:
To make additional tests make an `image_internal_test.go` file and put them in there.